package.json

```diff
@@ -13,7 +13,8 @@
   "resolutions": {
     "@expo/config-plugins": "^5.0.0",
     "strip-ansi": "6.0.0",
-    "string-width": "4.2.3"
+    "string-width": "4.2.3",
+    "react-native-svg": "13.9.0"
```

Comment:
I was just thinking it would be good to leave a comment here for why this resolution is needed, if we ever want to try to remove dependencies and inconsistencies from Resell in the future.

Reply:
Unfortunately this is a JSON file so we can't really include comments. I'll add it to this PR though.